document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java

```diff
@@ -361,7 +361,13 @@ private static StringBuilder prepareFilterStringForInOperator(
             .map(
                 val -> {
                   paramsBuilder.addObjectParam(val).addObjectParam(val);
-                  return "((jsonb_typeof(to_jsonb("+parsedExpression+")) = 'array' AND to_jsonb("+parsedExpression+") @> jsonb_build_array(?)) OR (jsonb_build_array(" + parsedExpression + ") @> jsonb_build_array(?)))";
+                  return "((jsonb_typeof(to_jsonb("
```

Comment:
Apologies for not catching it before. Can use `String.format()` and substitute placeholders of `%s` to improve readability.